platzky/seo/seo.py

```
@@ -1,11 +1,16 @@
+
 import typing as t
 import urllib.parse
 from os.path import dirname
 
-from flask import Blueprint, current_app, make_response, render_template, request
+from flask import Blueprint, Response, current_app, make_response, render_template, request
 
 
-def create_seo_blueprint(db, config: dict[str, t.Any], locale_func: t.Callable[[], str]):
+def create_seo_blueprint(
+    db: t.Any, config: dict[str, t.Any], locale_func: t.Callable[[], str]
```

Ruff (0.14.10)

Dynamically typed expressions (typing.Any) are disallowed in `db`

(ANN401)